runelite-client/src/main/java/net/runelite/client/plugins/corp/CorpConfig.java

```diff
@@ -35,10 +35,21 @@
 		keyName = "showDamage",
 		name = "Show damage overlay",
 		description = "Show total damage overlay",
-		position = 2
+		position = 0
 	)
 	default boolean showDamage()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+		keyName = "showDarkCore",
```

Comment:
To match the method name
```suggestion
		keyName = "markDarkCore",
```